README.md

```diff
@@ -154,6 +156,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+#### OS X
+
+To install the desktop entry for Alacritty, run
+
+```sh
+make app
```

Comment:
Wait...@jwilm are you saying that scrollback is coming? I thought it was a tmux or bust kind of situation